src/Table/Table.test.jsx

```diff
@@ -260,4 +278,25 @@ describe('<Table />', () => {
       expect(x2Spy).toBeCalledWith('desc');
     });
   });
+  describe('that is fixed', () => {
+    const wrapper = shallow(<Table {...fixedProps} />);
+
+    it('with col width classnames on headings', () => {
+      const tableHeadings = wrapper.find('th');
+
+      expect(tableHeadings).toHaveLength(fixedProps.columns.length);
+      expect(tableHeadings.at(0).hasClass('col-4')).toEqual(true);
+      expect(tableHeadings.at(1).hasClass('col-2')).toEqual(true);
+      expect(tableHeadings.at(2).hasClass('col-3')).toEqual(true);
+    });
+
+    it('with col width classnames on cells', () => {
```

Comment:
Would it be useful to add a test with column widths that doesn't have `tableFixed` equal to `true` to show that we shouldn't see the fixed behavior even if column widths are defined but `tableFixed` is `false`?

Reply:
:+1: added those tests [here](https://github.com/edx/paragon/pull/179/files#diff-57ba469286d69d3fbdda0a86dda12540R307).